src/plays/shopping-cart/reducer.tsx

```diff
@@ -0,0 +1,66 @@
+import { CartItem, InitialState } from "ShoppingCartTypes";
+
+export default function reducer(state: any, action: any) {

```

Comment:
The enum declared as per [this](https://github.com/reactplay/react-play/pull/779/files#r1016843253) comment can be leveraged to use proper types here for action. Also, an interface for the state would be beneficial. Something like below

```suggestion
export default function reducer(state: IState, action: REDUCER_ACTIONS) {

```